### PR TITLE
renamed scripts for clarity - fixed issue with reading token from reg…

### PR DESCRIPTION
**Pull request by matteoformica-amido**
- renamed scripts for clarity
- fixed issue with reading token from registration/resetpwd resume journeys
- improved logging
- removed unused scripts references

### CODE REVIEW

File: config/phase-0/am-scripts/scripts-content/ch-registration-verify-token.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+  if(errorFound){
+    if (callbacks.isEmpty()) {
+      action = fr.Action.send(
+          new fr.TextOutputCallback(
+              fr.TextOutputCallback.ERROR,
+              "Error While parsing token"
+          )
+      ).build()
+    }
+  } else if (Math.round(Difference_In_Time/(1000 * 60)) < 10080){
```
Comment (steveclewer-amido) [inline config/phase-0/am-scripts/scripts-content/ch-registration-verify-token.js:74]: This value doesn't match above?

File: config/phase-0/am-scripts/scripts-content/ch-password-reset-verify-token.js [hunk 2]
```diff
@@ -56,7 +49,7 @@ var fr = JavaImporter(
             )
         ).build()
       }
-    } else if (Math.round(Difference_In_Time/(1000 * 60)) < 10080){
+    } else if (Math.round(Difference_In_Time/(1000 * 60)) < 360){
```
[the remark this answers is not in the document]
Comment (matteoformica-amido) [reply config/phase-0/am-scripts/scripts-content/ch-password-reset-verify-token.js:52]: yes, token must be valid for 6 hours

**Review by steveclewer-amido — APPROVED** [review]

File: config/phase-0/am-scripts/scripts-content/ch-registration-verify-token.js (hunk 1 above)
Comment (matteoformica-amido) [reply config/phase-0/am-scripts/scripts-content/ch-registration-verify-token.js:74]: argh you're right

Comment (matteoformica-amido) [reply config/phase-0/am-scripts/scripts-content/ch-registration-verify-token.js:74]: the registration token must be valid for 7 days instead

**Pull request merged**